src/store/GirderAPI.ts

```diff
@@ -175,6 +94,26 @@ export default class GirderAPI {
     return result.data.length > 0 ? result.data[0] : null;
   }
 
+  async move(resources: IGirderSelectAble[], folderId: string) {
+    const resourceObj: { folder: string[]; item: string[] } = {
+      folder: [],
+      item: []
+    };
+    for (const resource of resources) {
+      const type = resource._modelType;
+      if (type === "folder" || type === "item") {
+        resourceObj[type].push(resource._id);
+      }
+    }
```

Comment:
```
resourceObj = resources.filter((resource) => {
  const type = resource._modelType;
  return type === "folder" || type === "item";
});
```
??

Reply:
`resourceObj` is an object containing two arrays of IDs, I can't build it using a filter
I could use a `forEach` instead of the `for ... of`, do you think it is worth it?

Reply:
Oh I see ! you can leave it this way I think